server/container/guice/cassandra-guice/src/main/java/org/apache/james/CassandraJamesServerMain.java

```diff
@@ -43,6 +44,7 @@
         new CassandraDomainListModule(),
         new CassandraRecipientRewriteTableModule(),
         new CassandraSieveRepositoryModule(),
+        new CassandraJmapModule(),
```

Comment:
I don't understand this message.
Creating a new Maven project ? Why ? What will be the benefits ? Why do you consider it wrong to have your Jmap data storage definition iside Cassandra project ? In what is it different from a UsersRepository, a DomainList or other interfaces used to store James data ?